src/test/cypress/cypress/integration/UserCard.spec.js

```diff
@@ -98,6 +98,52 @@ describe('User Card ', function () {
     })
   })
 
+  describe('Show automated response in preview for some user cards', function () {
+
+    it('Show automated response in preview for Confirmation user card if user enabled to respond', () => {
+
+      cy.loginOpFab('operator1_fr', 'test');
+      cy.get('#opfab-navbarContent').find('#opfab-newcard-menu').click();
+      cy.get("of-usercard").should('exist');
+      cy.get("#of-usercard-service-selector").find('select').select('User card examples');
+      cy.get("#of-usercard-process-filter").find('select').select('Examples for new cards 2');
+      cy.get("#opfab-state-filter").find('select').select('Confirmation');
+      cy.get('#opfab-recipients').click();
+      // Select recipent entity not in user entities
+      cy.get('#opfab-recipients').find('li').eq(0).click();
+      
+      cy.get('#opfab-recipients').click();
+      cy.get('#question').type('Confirm YES or NO');
+
+      cy.get('#opfab-usercard-btn-prepareCard').click();
+      cy.get("of-card-detail").should('exist');
+
+      // No responses shown 
+      cy.get("#childs-div").should('be.empty');
+      // Cancel sending
+      cy.get('#opfab-usercard-btn-refuse').click();
+
+      cy.get('#opfab-recipients').click();
+      // Select also one of user entities as recipent
+      cy.get('#opfab-recipients').find('li').eq(1).click();
+      cy.get('#opfab-recipients').click();
+
+      cy.get('#opfab-usercard-btn-prepareCard').click();
+      cy.get("of-card-detail").should('exist');
+
+      // Feed preview show response from current entity
+      cy.get('#opfab-feed-lightcard-hasChildCardFromCurrentUserEntity').should('exist');
+      // Card preview show responses list
+      cy.get("#childs-div").should('not.be.empty');
+      // Response table has 1 header and 1 row 
+      cy.get("#childs-div").find('tr').should('have.length', 2);
+      // Cancel sending
+      cy.get('#opfab-usercard-btn-refuse').click();
```

Comment:
In this test or in a new one  , send the card and check it is received in the feed open it and see response from child card is present